gateway/handlers/proxy.go

```diff
@@ -162,7 +162,7 @@ func invokeService(w http.ResponseWriter, r *http.Request, metrics metrics.Metri
 
 	defaultHeader := "text/plain"
 
-	w.Header().Set("Content-Type", GetContentType(response.Header, r.Header, defaultHeader))
+	w.Header().Set("Content-Type", GetContentType(response.Header, defaultHeader))
```

Comment:
Do we need this line anymore?

Reply:
I think it is necessary because `w.Header` needs to be set with the proper ContentType. If we don't do this, then the header won't be set with either the response header or the default.

Also, `clientHeader` defined above at line 160 appears to be created but not used:
https://github.com/openfaas/faas/pull/435/files/f1c825003d86b55242cb9e55e250d03fe4784ceb#diff-a7739e9c4834f2d0769169a1c27e7774R160